frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx

```diff
@@ -49,16 +49,31 @@ function Inventory({ bag }) {
               key="eggButton"
               type="button"
               className="eggButton"
-              onFocus={() => handleEnter()}
+              onFocus={() => handleEnter("Oeuf de quetzalcoatlus")}
               onBlur={handleLeave}
-              onMouseEnter={() => handleEnter()}
+              onMouseEnter={() => handleEnter("Oeuf de quetzalcoatlus")}
               onMouseLeave={handleLeave}
             >
               <img src={egg} className="eggImgInBag" alt="L'oeuf" />
             </button>
           ) : (
             ""
           )}
+          {bag.find((e) => e === "tooth") ? (
+            <button
+              key="toothButton"
+              type="button"
+              className="toothButton"
+              onFocus={() => handleEnter("Dent du Tyranossaurus Rex")}
+              onBlur={handleLeave}
+              onMouseEnter={() => handleEnter("Dent du T-Rex")}
```

Comment:
Why aren't the two pieces of information identical?

Reply:
corred i fixed it